x-pack/test/security_solution_cypress/es_archives/threat_data/data.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "doc",
+  "value": {
+    "id": "_eZE7mwBOpWiDweStB_c",
+    "index": "threat-data-001",
```

Comment:
`threat-data` is not very descriptive, especially when paired with `threat_indicator`. Perhaps we could call this `suspicious_source_events` or something like that?